homework/index.js

```diff
@@ -19,20 +19,49 @@
   function createAndAppend(name, parent, options = {}) {
     const elem = document.createElement(name);
     parent.appendChild(elem);
-    Object.entries(options).forEach(([key, value]) => {
+    Object.entries(options).forEach(([key, content]) => {
```

Comment:
In general, you should not make trivial changes such as this in a PR. Every change you make gets incorporated in the pull request and requires time from the reviewer to review. In this case it would be wasted time.